ds3_java_cli/src/main/java/com/spectralogic/ds3cli/Ds3Cli.java

```diff
@@ -104,7 +104,7 @@ public Ds3Cli(final Ds3Provider provider, final Arguments args, final FileUtils
         jsonViews.put(CommandValue.GET_JOB,                new com.spectralogic.ds3cli.views.json.GetJobView());
         jsonViews.put(CommandValue.DELETE_FOLDER,          deleteView);
         jsonViews.put(CommandValue.PUT_JOB,                new com.spectralogic.ds3cli.views.json.PutJobView());
-        jsonViews.put(CommandValue.GET_TAPES,              new com.spectralogic.ds3cli.views.json.GetTapesWithFullDetailsView());
+        jsonViews.put(CommandValue.GET_TAPES,              new com.spectralogic.ds3cli.views.json.GetTapesView());
```

Comment:
The command actually returns Tape, but it tries to parse it as a Detailed (and goes boom). I created a view to parse and display a Tape and was leaving the other in there if it is used elsewhere or implemented later.